tools/traffic/config/config.go

```diff
@@ -0,0 +1,137 @@
+package config
+
+import (
+	"errors"
+	"time"
+
+	"github.com/Layr-Labs/eigenda/common"
+	"github.com/urfave/cli"
+)
+
+// Config configures a traffic generator.
+type Config struct {
```

Comment:
We could reuse the configs from dependencies directly like we do [here](https://github.com/Layr-Labs/eigenda/blob/master/retriever/config.go#L16-L21)

Reply:
change made